xml/decoders/TCS_WOWDieselV5-121.xml

```diff
@@ -0,0 +1,12737 @@
+            <enumChoice choice="Gyra Light" value="7"/>
+            <enumChoice choice="Mars Light" value="2"/>
+            <enumChoice choice="Rotary Beacon (New Style)" value="69"/>
+            <enumChoice choice="Rotary Beacon (Old style)" value="6"/>
+            <enumChoice choice="Rule 17" value="8"/>
+            <enumChoice choice="Single Pulse Strobe" value="4"/>
+          </enumChoiceGroup>
+        </enumVal>
+        <label>              F0R (Yellow) Effect</label>
+      </variable>
+      <variable default="0" item="Output 3 effect generated" mask="VVXXVVVV" CV="53" minOut="3">
+        <value>0</value>
+        <enumVal>
+          <enumChoiceGroup xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
+            <enumChoice choice="Auto Mars" value="13"/>
+            <enumChoice choice="Brake Lights" value="14"/>
+            <enumChoice choice="Constant Bright" value="0"/>
+            <enumChoice choice="Constant Dim" value="12"/>
+            <enumChoice choice="Custom Light 1" value="70"/>
+            <enumChoice choice="Custom Light 2" value="71"/>
+            <enumChoice choice="Ditch Light Side 1" value="10"/>
+            <enumChoice choice="Ditch Light Side 2" value="11"/>
+            <enumChoice choice="Double Pulse Strobe" value="5"/>
+            <enumChoice choice="Firebox Flicker" value="1"/>
+            <enumChoice choice="Flashing F.R.E.D." value="3"/>
+            <enumChoice choice="Gyra Light" value="7"/>
+            <enumChoice choice="Mars Light" value="2"/>
+            <enumChoice choice="Rotary Beacon (New Style)" value="69"/>
+            <enumChoice choice="Rotary Beacon (Old style)" value="6"/>
+            <enumChoice choice="Rule 17" value="8"/>
+            <enumChoice choice="Single Pulse Strobe" value="4"/>
+          </enumChoiceGroup>
+        </enumVal>
+        <label>F3 (Brown) Effect</label>
+      </variable>
+      <variable default="0" item="Output 3 behavior" mask="XXVVXXXX" CV="53" minOut="3">
+        <enumVal>
+          <enumChoice choice="Function on running forward" value="0"/>
+          <enumChoice choice="Function on running reverse" value="1"/>
+          <enumChoice choice="Function on both directions" value="2"/>
+        </enumVal>
+        <label>F3 (Brown) Directionality</label>
+      </variable>
+      <variable default="0" item="Function 2 effect generated" mask="VVXXVVVV" CV="52" minOut="2">
+        <value>0</value>
+        <enumVal>
+          <enumChoiceGroup xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
+            <enumChoice choice="Auto Mars" value="13"/>
+            <enumChoice choice="Brake Lights" value="14"/>
+            <enumChoice choice="Constant Bright" value="0"/>
+            <enumChoice choice="Constant Dim" value="12"/>
+            <enumChoice choice="Custom Light 1" value="70"/>
+            <enumChoice choice="Custom Light 2" va
```

Comment:
This mask doesn't validate. I think what you intended was mask="XXXXXXXX VVVVVVVV" .

But that one is a bit bizarre because it degenerates to a decVal.